Sources/PrivacyDashboard/BrokenSiteReporting/ExpiryStorage.swift

```diff
@@ -21,22 +21,37 @@ import Persistence
 
 public typealias KeyValueStoringDictionaryRepresentable = KeyValueStoring & DictionaryRepresentable
 
+public struct ExpiryStorageConfiguration {
+
+    var expiryDatesStorageKey: String
+    var valueExpiryDateKey: String
+    var valueDataKey: String
+
+    public static let defaultConfig = ExpiryStorageConfiguration(
+        expiryDatesStorageKey: "com.duckduckgo.UserDefaultExpiryStorage",
+        valueExpiryDateKey: "com.duckduckgo.UserDefaultExpiryStorage.valueExpiryDate",
+        valueDataKey: "com.duckduckgo.UserDefaultExpiryStorage.valueData"
+    )
+
+    public static let autofillConfig = ExpiryStorageConfiguration(
```

Comment:
Created Autofill specific storage keys so that broken site reports and autofill breakage reports are stored separately